src/ReactiveUI/Activation.cs

```diff
@@ -315,16 +343,29 @@ static IDisposable handleViewModelActivation(IViewFor view, IObservable<bool> ac
 
     /// <summary>
     /// This class implements View Activation for classes that explicitly describe
-    /// their activation via ICanActivate. This class is used by the framework.
+    /// their activation via <see cref="ICanActivate"/>. This class is used by the framework.
     /// </summary>
     public class CanActivateViewFetcher : IActivationForViewFetcher
```

Comment:
Hmm, I don't know.  

My perspective on breaking stuff has changed since starting work on video courseware; there has to be an incredibly good reason to do it as it wastes the time of consumers and my time if the courseware needs to be updated. @kentcb probably has similar feelings as he's going through this process right now with authoring a book.

I had a quick look at the GitHub for Desktop aka Visual Studio plugin source code and it has taken no references to this class. It's one of the larger open-source projects that uses RxUI.

https://github.com/github/VisualStudio/search?utf8=%E2%9C%93&q=CanActivateViewFetcher